src/components/TextInput/index.js

```diff
@@ -14,6 +16,22 @@ class TextInput extends React.Component {
         if (this.props.name) {
             this.textInput.setAttribute('name', this.props.name);
         }
+
+        // Forcefully activate the soft keyboard when the user switches between tabs while input was focused.
+        this.unsubscribeVisibilityListener = Visibility.onVisibilityChange(() => {
```

Comment:
I think the naming is misleading here.

```suggestion
        this.removeVisibilityListener = Visibility.onVisibilityChange(() => {
```

Reply:
Fixed.